bin/strata-client/src/extractor.rs

```diff
@@ -575,7 +577,7 @@ mod tests {
             break;
         }
 
-        let mut rng = rand::thread_rng();
+        let mut rng = rand::rngs::OsRng;
```

Comment:
```suggestion
        let mut rng = OsRng;
```

Reply:
Also do we need this line at all? Can't we just pass `OsRng` to whatever was taking `rng` here?

Reply:
Maybe we can do `(&mut OsRng)`? Yeah that would be annoying also, big ugly and hard-to-visually-parse thing.

Reply:
Yeah, a separate assignment isn't particularly useful; the intent was just to minimize changes. I actually prefer `&mut OsRng` and will change.